### PR TITLE
Node mem and node io scenarios; Updating Litmus version

### PR DESCRIPTION
**Pull request by paigerube14**
### Description
Adding the node memory and node io stress scenario to litmus in the config
Updating litmus version to 1.13.6

### CODE REVIEW

File: scenarios/node_io_engine.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            ## Number of core of CPU
+            - name: CPU
+              value: '1'
+
+            ## Total number of workers default value is 4
+            - name: NUMBER_OF_WORKERS
+              value: '3'
+
+             ## enter the comma separated target nodes name
+            - name: TARGET_NODES
```
Comment (chaitanyaenr) [inline scenarios/node_io_engine.yaml:34]: @paigerube14 The scenario is working perfectly without any issues when not targeting specific nodes but it seems to fail when trying to hog a particular node. We might also want a way to log the error. Thoughts?

Comment (chaitanyaenr) [reply scenarios/node_io_engine.yaml:34]: Sure thing.

File: scenarios/node_mem_engine.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        components:
+          env:
+            # set chaos duration (in sec) as desired
+            - name: TOTAL_CHAOS_DURATION
+              value: '30'
+
+            ## Specify the size as percent of total node capacity Ex: '30'
+            ## Note: For consuming memory in mebibytes change the variable to MEMORY_CONSUMPTION_MEBIBYTES
+            - name: MEMORY_CONSUMPTION_PERCENTAGE
+              value: '300'
```
Comment (chaitanyaenr) [inline scenarios/node_mem_engine.yaml:24]: Maybe set this to 30% by default instead of 300?

**Review by chaitanyaenr — APPROVED** [review]
LGTM

File: scenarios/node_mem_engine.yaml (hunk 2 above)
Comment (paigerube14) [reply scenarios/node_mem_engine.yaml:24]: sure!

**Pull request merged**